src/components/app/routes/loaders/rootLoader.js

```diff
@@ -18,7 +19,8 @@ function fetchLicenseRequests({
   });
   const config = getConfig();
   const url = `${config.ENTERPRISE_ACCESS_BASE_URL}/api/v1/license-requests/?${queryParams.toString()}`;
-  return getAuthenticatedHttpClient().get(url);
+  const response = getAuthenticatedHttpClient().get(url);
+  return response.data;
```

Comment:
Opting to return `.data` in these functions rather than accessing the `.data` property in the `fetchBrowseAndRequestConfiguration` function below.